benchmarks/benchmarks/long/esmf_regridder.py

```diff
@@ -0,0 +1,176 @@
+"""Slower benchmarks for :mod:`esmf_regrid.esmf_regridder`."""
+
+from pathlib import Path
+
+import numpy as np
+import dask.array as da
+import iris
+from iris.cube import Cube
+
+from benchmarks import disable_repeat_between_setup
+from esmf_regrid.experimental.unstructured_scheme import (
+    GridToMeshESMFRegridder,
+    MeshToGridESMFRegridder,
+)
+from esmf_regrid.schemes import ESMFAreaWeightedRegridder
+from esmf_regrid.tests.unit.schemes.test__cube_to_GridInfo import _grid_cube
+
+
+class PrepareScalabilityGridToGrid:
+    params = [50, 100, 500, 1000, 2000]
```

Comment:
A Met Office specific acceptance criterion is to demonstrate regridding of...

>UM N1280 to equivalent LFRic res

Can a parameter value be put in to demonstrate this specifically?

Reply:
The current parameters have been chosen in such a way that they run on my machine with the allotted resources. I think expanding the parameters is worthwhile, though perhaps in another PR involving a way to allot more resources to the task.

Reply:
We have agreed offline to revisit larger parameterisation later.